File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashmap = {}
        for i in range(len(nums)):
            find = target-nums[i]
            if find in hashmap:
                return [i, hashmap[find]]
            hashmap[nums[i]]=i
        

File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer, Integer> hashMap = new HashMap<>();
        for(int i=0; i<nums.length; i++){
            int find = target-nums[i];
            if(hashMap.containsKey(find)){
                int[] ans = {i, hashMap.get(find)};
                return ans;
            }
            hashMap.put(nums[i], i);
        }
        return null;
    }
}